app/checks/free_ram.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import re
+from os import environ
+from sys import stderr
+
+if 'RAMPERCENT' not in environ:
```

Comment:
RAMPERCENT I would rename to `MAX_RAM_PERCENTAGE` to follow the convention of the disk-space check.